crypto/sleep.c

```diff
@@ -91,6 +92,9 @@ static void ossl_sleep_secs(uint64_t secs)
          */
         sleep(uint_times);
     }
+
+    /* Now, sleep the remaining seconds */
+    sleep((unsigned int)secs);
```

Comment:
This whole implementation seems a lot more complicated than something like

```
static void ossl_sleep_secs(uint64_t secs)
{
    while (secs > UINT_MAX) {
        sleep(UINT_MAX);
        secs -= UINT_MAX;
    }
    sleep((unsigned int)secs);
}
```

Reply:
Yes, and even this is quite ridiculous because who needs to sleep for more than UINT_MAX seconds? We do not support systems with less than 32 bit int/unsigned int.